acord/models/guild.py

```diff
@@ -94,7 +102,7 @@ class Guild(pydantic.BaseModel, Hashable):
     members: Dict[Snowflake, Member]
     """ Mapping of all members in guild """ 
 
-    mfa_level: int
+    mfa_level: MFALevel
```

Comment:
:class:\`MFALevel\`

Reply:
Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c